cli/features/tutorial.feature

```diff
@@ -134,7 +134,9 @@ Feature: Following the tutorial
           - 'mongo'
       """
     When running "exo setup" in this application's directory
+    And setting up a temporary mongo docker container
```

Comment:
We could do #81 instead, the only reason for this PR would be if we wanted to get #71 through more quickly and then remove this change later on when the changes in #81 are implemented.